insights/specs/default.py

```diff
@@ -478,7 +478,7 @@ def package_and_java(broker):
     rc_local = simple_file("/etc/rc.d/rc.local")
     redhat_release = simple_file("/etc/redhat-release")
     resolv_conf = simple_file("/etc/resolv.conf")
-    rhn_charsets = simple_command("/usr/bin/rhn-charsets")
+    rhn_charsets = first_of([simple_command("/usr/bin/rhn-charsets"), simple_file("database-character-sets")])
```

Comment:
The simple_file should go in sos_archive.py as
`rhn_charsets = simple_file("database-character-sets")`